internal/worker/api/api.go

```diff
@@ -3,3 +3,5 @@
 package api
 
 const BasePath = "/api/worker/v1"
+const CloudBasePath = "/api/composer-worker/v1"
+const OAuthURL="https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token"
```

Comment:
Should we make this configurable to work with staging/not only RH infra?

Reply:
absolutely